src/main/java/com/textmessenger/model/entity/dto/Token.java

```diff
@@ -0,0 +1,13 @@
+package com.textmessenger.model.entity.dto;
+
+import lombok.Data;
+
+@Data
+public class Token {
+  private String accessToken;
+  private static final String tokenType = "Bearer";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. ([squid:S00115](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS00115))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=142&issues=AWWFmk0iXKENQn1S911g&open=AWWFmk0iXKENQn1S911g)

[ISSUE_KEY]: AWWFmk0iXKENQn1S911g